frontend/src/pages/Search/SearchBar.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Multiselect from 'react-widgets/lib/Multiselect';
+
+import { useGetValidMunicipalities } from '../../sharedComponents/hooks/GetValidMunicipalities';
+
+const OuterWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0.5em;
+  & > * {
+    margin-right: 0.3em;
+  }
+`;
+
+const InputWrapper = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+`;
+
+const Select = styled(Multiselect)`
+  & > .rw-widget-picker > div {
+    display: flex;
+    align-items: center;
+    & > ul > li {
+      margin-top: 0px;
+    }
+  }
+`;
+
+export const SearchBar = ({ setQuery }) => {
+  const nameField = useRef('');
+  const descriptionField = useRef('');
```

Comment:
Why are we using refs and not state?

It adds unnecessary overhead as far as I can see as we never actually use the ref updates. This is better solved with onChange and useState if I am not missing something

Reply:
The main reason I chose useRef is that useState/onChange forces re-render on every input, which should really only be necessary when the search button is clicked anyways. I can change if you think the cons of useRef outweigh the pros.